### ISSUE TITLE
Emoji a11y

### ISSUE THREAD

**Issue by Zemke** [Zemke/just#24]
Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby

**Comment by Zemke** [comment]
This is done for UI emojis. Not planning this for messages.
